demo-terraform/aws/wordpress_ec2_alb_rds/servers.tf

```diff
@@ -31,61 +31,66 @@ EOT
 }
 
 resource "aws_key_pair" "kp" {
-    key_name   = "labkey-${random_string.lab_id.result}"
-    public_key = file("${var.aws_key_pair_pub}.pub")
+  key_name   = "labkey-${random_string.lab_id.result}"
+  public_key = file("${var.aws_key_pair_pub}.pub")
 }
 
 data "aws_ami" "wp_ami" {
-    most_recent = true
-    name_regex = "^.*ubuntu-focal-20.04-amd64-server-.*"    
-    owners = [ "099720109477" ] ## Canonical
-
-    filter {
-        name   = "root-device-type"
-        values = ["ebs"]
-    }
-
-    filter {
-        name   = "virtualization-type"
-        values = ["hvm"]
-    }
+  most_recent = true
+  name_regex  = "^.*ubuntu-focal-20.04-amd64-server-.*"
+  owners      = ["099720109477"] ## Canonical
+
+  filter {
+    name   = "root-device-type"
+    values = ["ebs"]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
 }
 
 ## Launch configuration for autoscaling
 resource "aws_launch_configuration" "lc_wp" {
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure IMDv1 is disabled for AWS EC2 instances in AWS Launch Configuration**
 
**Vulnerability**
AWS Launch Configuration that have IMDv1 enabled makes the end instances vulnerable to server side request forgery (SSRF) attacks.
 
**Remediation**
In AWS Console - 
 1. When launching a new instance in the Amazon EC2 console, select the following options on the Configure Instance Details page:
     a. Under Advanced Details, for Metadata accessible, select Enabled.
     b. For Metadata version, select V2.

source: accurics-